meta/documents/user_guide_en.md

```diff
@@ -3,17 +3,450 @@
+The plugin Elastic Export is required to use this format.
 
-Follow the instructions given on the [Exporting data](https://www.plentymarkets.co.uk/manual/data-exchange/exporting-data/#4) page of the manual to set up Google Shopping in plentymarkets.
+Refer to the [Exporting data formats for price search engines](https://knowledge.plentymarkets.com/en/basics/data-exchange/exporting-data#30) page of the manual for further details about the individual format settings.
 
-## License
+The following table lists details for settings, format settings and recommended item filters for the format **GoogleShopping-Plugin**.
+
+<table>
+    <tr>
+        <th>
+            Settings
+        </th>
+        <th>
+            Explanation
+        </th>
+    </tr>
+    <tr>
+        <td class="th" colspan="2">
+            Settings
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Format
+        </td>
+        <td>
+            Choose <b>GoogleShopping-Plugin</b>.
+        </td>        
+    </tr>
+    <tr>
+        <td>
+            Provisioning
+        </td>
+        <td>
+            Choose <b>URL</b>.
+        </td>        
+    </tr>
+    <tr>
+        <td>
+            File name
+        </td>
+        <td>
+            The file name must have the ending <b>.csv</b> for google shopping to be able to import the file successfully.
+        </td>        
+    </tr>
+    <tr>
+        <td class="th" colspan="2">
+            Item filter
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Active
+        </td>
+        <td>
+            Choose <b>active</b>.
+        </td>        
+    </tr>
+    <tr>
+        <td>
+            Markets
+        </td>
+        <td>
+            Choose one or multiple order referrer. The chosen order referrer has to be active at the variation for the item to be exported.
+        </td>        
+    </tr>
+    <tr>
+        <td class="th" colspan="2">
+            Format settings
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Order referrer
+        </td>
+        <td>
+            Choose the order referrer that should be assigned during the order import.
+        </td>        
+    </tr>
+    <tr>
+        <td>
+            Preview text
+        </td>
+        <td>
+            This option does not affect this format.
+        </td>        
+    </tr>
+    <tr>
+        <td>
+            Offer price
+        </td>
+        <td>
+            This option is not relevant for this format.
+        </td>        
+    </tr>
+    <tr>
+        <td>
+            VAT note
+        </td>
+        <td>
+            This option is not relevant for this format.
+        </td>        
+    </tr>
+</table>
+
+## 3 Overview of available columns
+
+<table>
+    <tr>
+        <th>
+            Column name
+        </th>
+        <th>
+            Explanation
+        </th>
+    </tr>
+    <tr>
+		<td>
+			id
+		</td>
+		<td>
+			<b>Content:</b> The google shopping <b>SKU</b> for the variation.
```

Comment:
Wäre hier vielleicht (im dt. muss es gleich sein) "The SKU for the variation which is linked to Google Shopping"?